client/cli_main.py

```diff
@@ -230,6 +266,72 @@ def main():
                         for friend in friends:
                             print(f"- {friend}")
                             print()
+                    elif sub_choice == "6":
+                        print()
+                        print("---Friends---")
+                        print()
+                        for friend in friends:
+                            print(f"- {friend}")
+                            print()
+                        friend_username = input("Enter the username of the friend you want to remove:")
+                        print()
+                        remove_user_friends(server_url, friend_username)
+                        #reloading friends object after removing a friend
+                        friends = get_user_friends(server_url)
+                    elif sub_choice == "7":
```

Comment:
Currently this is giving me this (error at the bottom):
```txt
PS D:\Programming\tmp\PeerBrain\client> python cli_main.py

Welcome to version 0.2 of the Brainwaves P2P client!
-------------------------------------------------------------------
The currently set server url is https://peerbrain.teckhawk.be/
-------------------------------------------------------------------

Please select what you want to do from the menu below.
1. Log in to the current server
2. Register account on the current server
3. Change server
4. Reset your password
Q to exit
>> 1

Logged in with valid token


MAIN MENU:


Please choose an option:

1. TECHNICAL ACTIONS
2. ACCOUNT ACTIONS
3. LOG OUT
Q to exit
>> 2

ACCOUNT MENU:


Please choose an option:

1. Check your account details
--------------------------------
2. Create a message
3. Show all messages from a friend
--------------------------------
4. Add a friend
5. Check friends list
6. Remove a friend from your friend list
--------------------------------
7. New thought display function test
B to return to main menu
>> 7

Please confirm your password to get your messages:


Please enter the username of the friend that you want to see messages from:

admin
Please choose a thought to read by entering its number or type B to go back:

1. TITLE: testing as admin       RATING: 1.2375035237499352
2. TITLE: testing message for nishant    RATING: 1.2375035237499352
3. TITLE: testing message while refactor cli     RATING: 2.1724833783268154
4. TITLE: The meaning of life?   RATING: 1.2375035237499352

Enter thought number: 4

TITLE:  The meaning of life?
RATING: 1.2375035237499352

MESSAGE:  You thought there would be something here, didn't you :-P.


Traceback (most recent call last):
  File "D:\Programming\tmp\PeerBrain\client\cli_main.py", line 360, in <module>
    main()
  File "D:\Programming\tmp\PeerBrain\client\cli_main.py", line 325, in main
    update_rating_for_thought(server_url, str(selected_thought["key"]))
KeyError: 'key'
NativeCommandExitException: Program "python.exe" ended with non-zero exit code: 1.
```

Reply:
It is already fixed in the server code. The get thoughts function did not return the key initially. I've made changes that do return it, but you need to run the dev server. You are testing this on main, which does not have the altered server code yet.

Reply:
My bad, will test on dev